examples/foomedical/src/pages/ChatsPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { Divider, Stack, Table, Title } from '@mantine/core';
+import { formatGivenName, formatFamilyName } from '@medplum/core';
+import { Patient, Practitioner, HumanName } from '@medplum/fhirtypes';
+import { Document, useMedplum, useMedplumProfile } from '@medplum/react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Loading } from '../components/Loading';
+import classes from './ChatsPage.module.css';
+
+export function Chats(): JSX.Element {
```

Comment:
I'm not fully happy with this component name, feel free to suggest a different one if you have a better idea

Reply:
```suggestion
export function MessageTable(): JSX.Element {
```